### DISCUSSION TITLE
Aborting job results in "connect: no such file or directory"

### DISCUSSION THREAD

**Discussion by ChristophSchmidpeter** [zrepl/zrepl#837] [Q&A] [answered]
I tried to abort a runnig backup job via
`$sudo zrepl signal reset backup_job `                                                                  
But it results into
`Post "http://unix/signal": dial unix /var/run/zrepl/control: connect: no such file or directory`

**Comment by problame** [comment, marked as answer]
Sorry for the late reply.

This error would occur if the daemon is not running or configured to listen on a non-default `control` socket.
